frontend/ui/widget/textviewer.lua

```diff
@@ -80,11 +80,11 @@ function TextViewer:init()
         }
     end
 
-    local title_text = TextWidget:new{
+    local title_text = TextBoxWidget:new{
         text = self.title,
         face = self.title_face,
         bold = true,
-        width = self.width - 2*self.title_padding - 2*self.title_margin,
+        width = self.width * 0.9,
```

Comment:
I just realized I might've been a bit cryptic. I mean something like

```lua
width = self.width - 2*self.title_padding - 2*self.title_margin - close_button_width,
```

where you'd get the CloseButton width by calling getSize().w on a CloseButton object.